senior-knights/course-schedulizer
# Export converted csv

This branch:
1. Makes the import button only parse the CSV into a `scheduleJSON`, save the `scheduleJSON` in the context, and log it to the console (ie. it no longer logs the JSON converted back to CSV).
2. Makes the export button work (ie. it will save a file called `schedule.csv` to your downloads with the context's `scheduleJSON` converted back into a CSV).
3. Makes the hamburger menu open to the right of the hamburger button (such that you can close the menu by clicking the hamburger button again), makes the import and export menu item buttons clickable for the entire width and height of the menu item (previously only the text portion was clickable, so there was a boarder that looked clickable but wouldn't trigger anything), and finally makes the buttons have the click icon when hovered (previously the label would only give the default mouse icon when hovered).

To Test:
1. `npm install` and `npm start` like usual.
2. Try out importing a CSV, and then exporting it.
3. Make sure that the changes to the import/export button styling make sense/look nice.

Closes #20
